libraries/helper.rb

```diff
@@ -74,9 +74,9 @@ def windows_cleanpath(path)
       end
 
       def application_cleanname(application_name)
-        if !application_name.include?('/') && !application_name.end_with?('/')
+        if application_name.count('/') == 0
           "#{application_name}/"
-        elsif application_name.chomp('/').include?('/') && application_name.end_with?('/')
+        elsif application_name.count('/') > 1
```

Comment:
same as previous, if the name with the last / removed still includes a / and the name ends in a / then remove the last /

Default Web Site/my app/ => Default Web Site/my app